enacts/calc.py

```diff
@@ -619,6 +619,42 @@ def daily_tobegroupedby_season(
     return daily_tobegroupedby_season
 
 
+def seasonal_groups(
+    time_coord, start_day, start_month, end_day, end_month
+):
+    """Create seasonal groups identified by first day of each season"""
```

Comment:
I complied to the comment, but I am also thinking that for the final result, we might want to output more than the season start... possibly the end too, so that a user can have the full seasonal information... it depends how we opt to describe this kind of objects.